src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java

```diff
@@ -0,0 +1,143 @@
+package org.folio.circulation.domain.policy.lostitem;
+
+import static org.folio.circulation.domain.policy.Period.from;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.folio.circulation.domain.policy.Period;
+import org.joda.time.DateTime;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.builders.LostItemFeePolicyBuilder;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class LostItemPolicyTest {
+
+  @Test
+  @Parameters( {
+    "Minutes, 78",
+    "Hours, 9",
+    "Days, 66",
+    "Weeks, 23",
+    "Months, 13",
+  })
+  public void shouldNotAgeItemToLostIfDueDateAfterNow(String interval, int duration) {
+    final Period period = from(duration, interval);
+    final LostItemPolicy lostItemPolicy = lostItemPolicyWithAgePeriod(period);
+
+    assertFalse(lostItemPolicy.canAgeLoanToLost(now(UTC)));
+  }
+
+  @Test
+  @Parameters( {
+    "Minutes, 43",
+    "Hours, 12",
+    "Days, 29",
+    "Weeks, 1",
+    "Months, 5",
+  })
+  public void shouldAgeItemToLostIfDueDateBeforeNow(String interval, int duration) {
```

Comment:
Am I correct in understanding that the due date has to be more in the past than the age period?

```suggestion
  public void shouldAgeItemToLostIfDueDateIsSufficientlyInThePast(String interval, int duration) {
```

Reply:
> Am I correct in understanding that the due date has to be more in the past than the age period?

Yes, the `dueDate + period >= now()`.

This particular test verifies the `dueDate + period > now()` case, there is a case that verifies `dueDate + period == now()`

Reply:
Thank you for explaining that. I think the name of the test needs to describe how the age to lost period has not passed, only referring to now gives me the impression that this is about the due date passing now.

Does that make sense?

Reply:
Ok, I'll use the name that you've suggested.

Reply:
Feel free to use another name. I think the important thing is that the name makes it clear that this is about the gap between the due date and now in relation to the age to lost period.